drivers/classic/src/display_current_settings.c

```diff
@@ -85,12 +85,12 @@ display_current_settings(int mode)
         fprintf(LOG_DEST, "EQUAL_AREA\t\tFALSE\n");
     }
     fprintf(LOG_DEST, "RESOLUTION\t\t%f\n", global_param.resolution);
-    fprintf(LOG_DEST, "TIME_STEP\t\t%d\n", global_param.dt);
-    fprintf(LOG_DEST, "SNOW_STEP\t\t%d\n", options.SNOW_STEP);
+    fprintf(LOG_DEST, "TIME_STEP\t\t%f\n", global_param.dt);
+    fprintf(LOG_DEST, "SNOW_STEP\t\t%f\n", global_param.snow_dt);
```

Comment:
I'm going to leave these as is.  `%f` and `%lf` are both allowed by `fprintf()` but have identical behaviors.  `float`s and promoted to `double`s automatically.  The same goes for `unsigned` and `short` `int`s which are promoted to `int`.  Note, this is only true for `fprintf()`, the correct format specifier is important for reader functions like `scanf()`.

Reference: http://stackoverflow.com/questions/4264127/correct-format-specifier-for-double-in-printf